src/pages/ALLO_ManageAllocations.page

```diff
@@ -155,7 +155,7 @@
                 parentRecordAction="/{!parentId}" header="{!pageTitle}" icon="custom_120" iconCategory="standard"
                 cancelAction="{!cancel}" saveAction="{!saveClose}" saveDisabled="{!opp<>null&&parentAmount==0}"/>
             <div> 
-                <apex:pageMessages id="messages"/>
+                <c:UTIL_PageMessages id="messages"/>
```

Comment:
I'm seeing messages in the new format, however if the [Manage Allocations] button is clicked on an Opportunity with a $0 or null Amount field, the page is displayed without any message. The Save button is disabled (which is good).

Reply:
@force2b I do not see anything in the controller that would add that error message, I agree that would be good to have but wondering if it should be done with this work?

Reply:
I'm pretty sure it was there earlier when we were testing on hangouts earlier. I cleared the amount field, and then clicked the Manage Allocations button. I could swear there was a message displayed on the screen. Could be wrong too, but I really thought it was there.

Reply:
I wonder if it was because you already had allocations saved and then nulled out the amount and then went back to manage?

Reply:
Just put the <apex:pageMessages> tag back in and retested. The message displayed is "You can only create allocations for opportunities with a positive amount."

Reply:
@force2b ok, I will take another look, thanks!

Reply:
@force2b thanks again for catching that! Chicken before the egg type of an issue, should be all set now!